packs/journals/classes.json

```diff
@@ -477,7 +477,7 @@
started with, you gain an ancestry feat at 5th level and every 4 levels thereafter.</p><h2>Reflex Expertise<span style=\"float:right\">Level 5</span></h2><p>Your reflexes are lightning fast. Your proficiency rank for Reflex saves increases to expert.</p><h2>Expert Spellcaster<span style=\"float:right\">Level 7</span></h2><p>Extended practice of the arcane has improved your capabilities. Your proficiency ranks for arcane spell attack rolls and spell DCs increase to expert.</p><h2>Magical Fortitude<span style=\"float:right\">Level 9</span></h2><p>Magical power has improved your body's resiliency. Your proficiency rank for Fortitude saves increases to expert.</p><h2>Perception Expertise<span style=\"float:right\">Level 11</span></h2><p>You remain alert to threats around you. Your proficiency rank for Perception increases to expert.</p><h2>Weapon Expertise<span style=\"float:right\">Level 11</span></h2><p>Through sheer experience, you've improved your technique with your weapons. Your proficiency ranks for simple weapons and unarmed attacks increase to expert.</p><h2>Defensive Robes<span style=\"float:right\">Level 13</span></h2><p>The flow of magic and your training combine to help you avoid attacks. Your proficiency rank in unarmored defense increases to expert.</p><h2>Weapon Specialization<span style=\"float:right\">Level 13</span></h2><p>You've learned how to inflict greater injuries with the weapons you know best. You deal an additional 2 damage with weapons and unarmed attacks in which you are an expert. This damage increases to 3 if you're a master, and 4 if you're legendary.</p><h2>Master Spellcaster<span style=\"float:right\">Level 15</span></h2><p>You command superlative spellcasting ability. Your proficiency ranks for arcane spell attack rolls and spell DCs increase to master.</p><h2>Prodigious Will<span style=\"float:right\">Level 17</span></h2><p>Your mind is simply too complex and sophisticated to be tampered with. Your proficiency rank for Will savesincreases to master. When you roll a success at a Will save, you get a critical success instead.</p><h2>Archwizard's Spellcraft<span style=\"float:right\">Level 19</span></h2><p>You command the most potent arcane magic and can cast a spell of truly incredible power. You gain a single 10th-level spell slot and can prepare a spell in that slot using arcane spellcasting. You can't use this spell slot for abilities that let you cast spells without expending spell slots or that give you more spell slots. Unlike with other spell slots, you don't gain more 10th-level spells as you level up, though you can take the @UUID[Compendium.pf2e.feats-srd.Item.Archwizard's Might] feat to gain a second slot.</p><h2>Legendary Spellcaster<span style=\"float:right\">Level 19</span></h2><p>You are a consummate spellcaster, with a comprehensive understanding of both arcane theory and practical spellcraft. Your proficiency ranks for arcane spell attack rolls and spell DCs increase to legendary.</p><p></p><p></p>",
```

Comment:
```suggestion
                "content": "<p><em>You are an eternal student of the secrets of the universe, using your mastery of magic to cast powerful spells. You treat magic like a science, cross-referencing the latest texts on practical spellcraft with ancient tomes to discover and understand arcane magic. Yet magical theory is vast, and there's no way you can study it all. Most wizards learn through formal schooling, with their curriculum informing a specific rubric, although particularly driven researchers sometimes piece together their own theories.</em></p><p><strong>Key Attribute: INTELLIGENCE</strong></p><p>At 1st level, your class gives you an ability boost to Intelligence.</p><p><strong>Hit Points: 6 plus your Constitution modifier</strong></p><p>You increase your maximum number of HP by this number at 1st level and every level thereafter.</p><h1>Roleplaying the Wizard</h1><h2>During Combat Encounters...</h2><p>You likely try to stay out of the fray, carefully judging when to use your spells. You save your most powerful magic to incapacitate threatening foes and use your cantrips when only weaker foes remain. When enemies pull out tricks like <em>@UUID[Compendium.pf2e.spells-srd.Item.Invisibility]</em> or <em>@UUID[Compendium.pf2e.spells-srd.Item.Fly]</em>, you answer with spells like <em>@UUID[Compendium.pf2e.spells-srd.Item.Revealing Light]</em> or <em>@UUID[Compendium.pf2e.spells-srd.Item.Earthbind]</em>, leveling the field for your allies.</p><h2>During Social Encounters...</h2><p>You provide a well of knowledge about arcane matters and solve arguments with logic.</p><h2>While Exploring...</h2><p>You locate magical auras and determine the arcane significance of magical writing or phenomena you uncover. When you run across an unusual obstacle to further exploration, you probably have a scroll that will make it easier to overcome.</p><h2>In Downtime...</h2><p>You learn new spells, craft magic items, or scribe scrolls for your party, and seek out new and exciting formulas in addition to spells. You might even forge scholarly connections and establish a school or guild of your own.</p><h2>You Might...</h2><ul><li>Have an unquenchable intellectual curiosity about how everything in the world around you works-magic in particular.</li><li>Believe fervently that your school of magic is superior (if you're a specialist) or that true mastery of magic requires knowledge of all schools (if you're a universalist).</li><li>Use esoteric jargon and technical terms to precisely describe the minutiae of magical effects, even though the difference is probably lost on other people.</li></ul><h2>Others Probably...</h2><ul><li>Consider you to be incredibly powerful and potentially dangerous.</li><li>Fear what your magic can do to their minds, bodies, and souls, and ask that you avoid casting spells in polite company, as few can identify whether one of your spells is harmless or malevolent until it's too late.</li><li>Assume you can easily solve all their problems, from dangerous weather to poor crop yields, and ask you for spells that can help them get whatever they desire.</li></ul><h1>Initial Proficiencies</h1><p>At 1st level, you gain the listed proficiency ranks in the following statistics. You are untrained in anything not listed unless you gain a better proficiency rank in some other way.</p><h2>Perception</h2><p>Trained in Perception</p><h2>Saving Throws</h2><p>Trained in Fortitude</p><p>Trained in Reflex</p><p>Expert in Will</p><h2>Skills</h2><p>Trained in Arcana</p><p>Trained in a number of additional skills equal to 2 plus your Intelligence modifier</p><h2>Attacks</h2><p>Trained in simple weapons</p><p>Trained in unarmed attacks</p><h2>Defenses</h2><p>Untrained in all armor</p><p>Trained in unarmored defense</p><h2>Spells</h2><p>Trained in spell attack modifier</p><p>Trained in spell DC</p><h2>Class DC</h2><p>Trained in wizard class DC</p><h1>Class Features</h1><p>You gain these features as a Wizard. Abilities gained at higher levels list the levels at which you gain them next to the features' names.</p><table class=\"pf2-table\"><thead><tr><th>Your Level</th><th>Class Features</th></tr></thead><tbody><tr><td>1</td><td>Ancestry and background, initial proficiencies, arcane spellcasting, arcane school, arcane bond, arcane thesis</td></tr><tr><td>2</td><td>Skill feat, wizard feat</td></tr><tr><td>3</td><td>2nd-rank spells, general feat, skill increase</td></tr><tr><td>4</td><td>Skill feat, wizard feat</td></tr><tr><td>5</td><td>3rd-rank spells, attribute boosts, ancestry feat, reflex expertise, skill increase</td></tr><tr><td>6</td><td>Skill feat, wizard feat</td></tr><tr><td>7</td><td>4th-rank spells, expert spellcaster, general feat, skill increase</td></tr><tr><td>8</td><td>Skill feat, wizard feat</td></tr><tr><td>9</td><td>5th-rank spells, ancestry feat, magical fortitude, skill increase</td></tr><tr><td>10</td><td>Attribute boosts, skill feat, wizard feat</td></tr><tr><td>11</td><td>6th-rank spells, general feat, perception expertise, skill increase, wizard weapon expertise</td></tr><tr><td>12</td><td>Skill feat, wizard feat</td></tr><tr><td>13</td><td>7th-rank spells, ancestry feat, defensive robes, skill increase, weapon specialization</td></tr><tr><td>14</td><td>Skill feat, wizard feat</td></tr><tr><td>15</td><td>8th-rank spells, attribute boosts, general feat, master spellcaster, skill increase</td></tr><tr><td>16</td><td>Skill feat, wizard feat</td></tr><tr><td>17</td><td>9th-rank spells, ancestry feat, prodigious will, skill increase</td></tr><tr><td>18</td><td>Skill feat, wizard feat</td></tr><tr><td>19</td><td>Archwizard's spellcraft, general feat, legendary archwizard, skill increase</td></tr><tr><td>20</td><td>Attribute boosts, skill feat, wizard feat</td></tr></tbody></table><h2>Ancestry and Background</h2><p>In addition to the abilities provided by your class at 1st level, you have the benefits of your selected ancestry and background.</p><h2>Attribute Boosts</h2><p>In addition to what you get from your class at 1st level, you have four free boosts to different attribute modifiers.</p><p>At 5th level and every 5 levels thereafter, you get four free boosts to different attribute modifiers. If an attribute modifier is already +4 or higher, it takes two boosts to increase it; you get a partial boost, and must boost that attribute again at a later level to increase it by 1.</p><h2>Initial Proficiencies</h2><p>At 1st level, you gain a number of proficiencies that represent your basic training. These proficiencies are noted at the start of this class.</p><h2>Wizard Spellcasting</h2><p>Through dedicated study and practice, you can harness arcane power to cast spells. You can cast arcane spells using the Cast a Spell activity, and you can supply material, somatic, and verbal components when casting spells.</p><p>At 1st level, you can prepare up to two 1st-rank spells and five cantrips each morning from the spells in your spellbook (see below), as well as one extra curriculum cantrip and one extra curriculum spell of each rank you can cast from your arcane school. Prepared spells remain available to you until you cast them or until you prepare your spells again. The number of spells you can prepare is called your spell slots.</p><p>As you increase in level as a wizard, your number of spell slots and the highest level of spells you can cast from spell slots increase, shown in the table below.</p><p>Some of your spells require you to attempt a spell attack roll to see how effective they are, or have your enemies roll against your spell DC (typically by attempting a saving throw). Since your key ability is Intelligence, your spell attack rolls and spell DCs use your Intelligence modifier.</p><h3><strong>Heightening Spells</strong></h3><p>When you get spell slots of 2nd level and higher, you can fill those slots with stronger versions of lower-level spells. This increases the spell's level, heightening it to match the spell slot. Many spells have specific improvements when they are heightened to certain levels.</p><h3><strong>Cantrips</strong></h3><p>A cantrip is a special type of spell that doesn't use spell slots. You can cast a cantrip at will, any number of times per day. A cantrip is always automatically heightened to half your level rounded up-this is usually equal to the highest level of spell you can cast as a wizard. For example, as a 1st-level wizard, your cantrips are 1st-level spells, and as a 5th-level wizard, your cantrips are 3rd-level spells.</p><h3><strong>Spellbook</strong></h3><p>Every arcane spell has a written version, usually recorded in a spellbook. You start with a spellbook worth 10 sp or less, which you receive for free and must study to prepare your spells each day. The spellbook contains your choice of 10 arcane cantrips and five 1st-level arcane spells. You choose these from the common spells on the arcane spell list or from other arcane spells you gain access to. Your spellbook's form and name are up to you. It might be a musty, leather-bound tome or an assortment of thin metal disks connected to a brass ring; its name might be esoteric, like <em>The Crimson Libram</em>, or something more academic, like <em>A Field Study in Practical Transmutation</em>.</p><p>Each time you gain a level, you add two arcane spells to your spellbook, of any level for which you have spell slots. You can also use the Arcana skill to add other spells that you find in your adventures.</p><table class=\"pf2-table\"><thead><tr><th>Your Level</th><th>Cantrips</th><th>1st</th><th>2nd</th><th>3rd</th><th>4th</th><th>5th</th><th>6th</th><th>7th</th><th>8th</th><th>9th</th><th>10th</th></tr></thead><tbody><tr><td>1</td><td>5</td><td>2</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>2</td><td>5</td><td>3</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>3</td><td>5</td><td>3</td><td>2</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>4</td><td>5</td><td>3</td><td>3</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>5</td><td>5</td><td>3</td><td>3</td><td>2</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>6</td><td>5</td><td>3</td><td>3</td><td>3</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>7</td><td>5</td><td>3</td><td>3</td><td>3</td><td>2</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>8</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>9</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>2</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>10</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>11</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>2</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>12</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>13</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>2</td><td>-</td><td>-</td><td>-</td></tr><tr><td>14</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>-</td><td>-</td><td>-</td></tr><tr><td>15</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>2</td><td>-</td><td>-</td></tr><tr><td>16</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>-</td><td>-</td></tr><tr><td>17</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>2</td><td>-</td></tr><tr><td>18</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>-</td></tr><tr><td>19</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>1*</td></tr><tr><td>20</td><td>5</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>1*</td></tr><tr><td style=\"color:white;width:120px;background-color:#522e2c;text-align:center;height:25px\" colspan=\"12\">* The Archwizard's Spellcraft class feature gives you a 10th-level spell slot that works a bit differently from other spell slots.</td></tr></tbody></table><h2>Arcane Thesis</h2><p>During your studies to become a full-fledged wizard, you produced a thesis of unique magical research on one of a variety of topics. You gain a special benefit depending on the topic of your thesis research. The arcane thesis topics presented in this book are below; your specific thesis probably has a much longer and more technical title like \"On the Methods of Spell Interpolation and the Genesis of a New Understanding of the Building Blocks of Magic.\"</p><p>Arcane theses can be found here:</p><ul><li>@UUID[Compendium.pf2e.classfeatures.Item.Experimental Spellshaping],</li><li>@UUID[Compendium.pf2e.classfeatures.Item.Improved Familiar Attunement],</li><li>@UUID[Compendium.pf2e.classfeatures.Item.Spell Blending],</li><li>@UUID[Compendium.pf2e.classfeatures.Item.Spell Substitution], and</li><li>@UUID[Compendium.pf2e.classfeatures.Item.Staff Nexus]</li></ul><h2>Arcane School</h2><p>Most wizards acquire their knowledge of spells from a formal educational institution, such as the Arcanamirium or the Magaambya. At 1st level, you choose your arcane school, which grants you magical abilities. </p><p>You gain additional spells and spell slots from the curriculum taught at the school you attended. Some wizards follow the school of unified magical theory, which attempts to forge a new school by studying independently and drawing information from a multitude of texts and tutors. Though a wizard with this approach lacks the focus of formal training, they have greater flexibility.</p><p>Your arcane school is where you devoted yourself to the study of spellcraft. Whether you learned in the storied halls of a formal institution, via an apprenticeship with an archmage, or taught yourself from secondhand tomes, your arcane school indelibly set the curriculum and direction of your magic. </p><p><strong>Curriculum Spells:</strong> You automatically add some of the spells listed in your school's curriculum to your spellbook. At 1st level, you add a cantrip and two 1st-rank spells of your choice. As soon as you gain the ability to cast wizard spells of a new rank, choose one of the spells from your curriculum of that rank to add to your spellbook. A superscript \"U\" indicates an uncommon spell. Your GM might allow you to swap or add other spells to your curriculum if they strongly fit the theme. </p><p><strong>Spell Slots:</strong> Each day, you can prepare an extra cantrip from your curriculum. You also gain an extra spell slot at each spell rank for which you have wizard spell slots. You can prepare only spells from your school's curriculum in these extra slots. Any spell listed in your curriculum of a suitable spell rank is eligible to be prepared in each of these extra slots, regardless of how you added the spell to your spellbook. </p><p><strong>School Spells:</strong> School spells are a type of focus spell. It costs 1 Focus Point to cast a focus spell, and you start with a focus pool of 1 Focus Point. You refill your focus pool during your daily preparations, and you can regain 1 Focus Point by spending 10 minutes using the Refocus activity to study your spellbook or conduct arcane research. </p><p>Focus spells are automatically heightened to half your level rounded up, much like cantrips. Focus spells don't require spell slots, and you can't cast them using spell slots. Certain feats give you more focus spells. The maximum Focus Points your focus pool can hold is equal to the number of focus spells you have, but it can never be more than 3 points. </p><p>You learn the initial school spell, which is unique to your school. You can learn the advanced school spell with the Advanced School Spell feat.</p><p>@UUID[Compendium.pf2e.classfeatures.Item.School of Ars Grammatica]</p><p>@UUID[Compendium.pf2e.classfeatures.Item.School of Battle Magic]</p><p>@UUID[Compendium.pf2e.classfeatures.Item.School of the Boundary]</p><p>@UUID[Compendium.pf2e.classfeatures.Item.School of Civic Wizardry]</p><p>@UUID[Compendium.pf2e.classfeatures.Item.School of Mentalism]</p><p>@UUID[Compendium.pf2e.classfeatures.Item.School of Protean Form]</p><p>@UUID[Compendium.pf2e.classfeatures.Item.School of Unified Magical Theory]</p><h2>Arcane Bond</h2><p>You place some of your magical power in a bonded item. Each day when you prepare your spells, you can designate a single item you own as your bonded item. This is typically an item associated with spellcasting, such as a wand, ring, or staff, but you are free to designate a weapon or other item. You gain the Drain Bonded Item free action.</p><h3>@UUID[Compendium.pf2e.actionspf2e.Item.Drain Bonded Item] <span class=\"pf2-icon\">F</span></h3><p><strong>Frequency</strong> once per day</p><p><strong>Requirements</strong> You haven't acted yet on your turn</p><hr /><p>You expend the power stored in your bonded item. During your turn, you gain the ability to cast one spell you prepared today and already cast, without spending a spell slot. You must still Cast the Spell and meet the spell's other requirements.</p><h2>Skill Feats<span style=\"float:right\">Level 2</span></h2><p>At 2nd level and every 2 levels thereafter, you gain a skill feat. You must be trained or better in the corresponding skill to select a skill feat.</p><h2>Wizard Feats<span style=\"float:right\">Level 2</span></h2><p>At 2nd level and every even-numbered level thereafter, you gain a wizard class feat.</p><h2>General Feats<span style=\"float:right\">Level 3</span></h2><p>At 3rd level and every 4 levels thereafter, you gain a general feat.</p><h2>Skill Increases<span style=\"float:right\">Level 3</span></h2><p>At 3rd level and every 2 levels thereafter, you gain a skill increase. You can use this increase either to increase your proficiency rank to trained in one skill you're untrained in, or to increase your proficiency rank in one skill in which you're already trained to expert.</p><p>At 7th level, you can use skill increases to increase your proficiency rank to master in a skill in which you're already an expert, and at 15th level, you can use them to increase your proficiency rank to legendary in a skill in which you're already a master.</p><h2>Ancestry Feats<span style=\"float:right\">Level 5</span></h2><p>In addition to the ancestry feat you started with, you gain an ancestry feat at 5th level and every 4 levels thereafter.</p><h2>Reflex Expertise<span style=\"float:right\">Level 5</span></h2><p>Your reflexes are lightning fast. Your proficiency rank for Reflex saves increases to expert.</p><h2>Expert Spellcaster<span style=\"float:right\">Level 7</span></h2><p>Extended practice of the arcane has improved your capabilities. Your proficiency ranks for arcane spell attack rolls and spell DCs increase to expert.</p><h2>Magical Fortitude<span style=\"float:right\">Level 9</span></h2><p>Magical power has improved your body's resiliency. Your proficiency rank for Fortitude saves increases to expert.</p><h2>Perception Expertise<span style=\"float:right\">Level 11</span></h2><p>You remain alert to threats around you. Your proficiency rank for Perception increases to expert.</p><h2>Weapon Expertise<span style=\"float:right\">Level 11</span></h2><p>Through sheer experience, you've improved your technique with your weapons. Your proficiency ranks for simple weapons and unarmed attacks increase to expert.</p><h2>Defensive Robes<span style=\"float:right\">Level 13</span></h2><p>The flow of magic and your training combine to help you avoid attacks. Your proficiency rank in unarmored defense increases to expert.</p><h2>Weapon Specialization<span style=\"float:right\">Level 13</span></h2><p>You've learned how to inflict greater injuries with the weapons you know best. You deal an additional 2 damage with weapons and unarmed attacks in which you are an expert. This damage increases to 3 if you're a master, and 4 if you're legendary.</p><h2>Master Spellcaster<span style=\"float:right\">Level 15</span></h2><p>You command superlative spellcasting ability. Your proficiency ranks for arcane spell attack rolls and spell DCs increase to master.</p><h2>Prodigious Will<span style=\"float:right\">Level 17</span></h2><p>Your mind is simply too complex and sophisticated to be tampered with. Your proficiency rank for Will savesincreases to master. When you roll a success at a Will save, you get a critical success instead.</p><h2>Archwizard's Spellcraft<span style=\"float:right\">Level 19</span></h2><p>You command the most potent arcane magic and can cast a spell of truly incredible power. You gain a single 10th-level spell slot and can prepare a spell in that slot using arcane spellcasting. You can't use this spell slot for abilities that let you cast spells without expending spell slots or that give you more spell slots. Unlike with other spell slots, you don't gain more 10th-level spells as you level up, though you can take the @UUID[Compendium.pf2e.feats-srd.Item.Archwizard's Might] feat to gain a second slot.</p><h2>Legendary Spellcaster<span style=\"float:right\">Level 19</span></h2><p>You are a consummate spellcaster, with a comprehensive understanding of both arcane theory and practical spellcraft. Your proficiency ranks for arcane spell attack rolls and spell DCs increase to legendary.</p><p></p><p></p>",
```